src/workers/forum_automod.py

```diff
@@ -24,70 +20,27 @@ async def autodelete_threads(self):
         return
 
     for channel_id, duration in settings.automod_inactivity.items():
-        num_removed = 0
-        num_error = 0
+        results = AutomodResults()
         channel = guild.get_channel(channel_id)
         if channel is None:
             continue
 
-        for thread in channel.threads:
-            if thread.flags.pinned:
-                # skip the for loop if the thread is pinned
-                continue
+        async for thread in channel.archived_threads(limit=None):
+            await automod_thread(self, channel_id, thread, duration, results)
```

Comment:
i think we can combine the archived_threads and threads together and just loop once

Reply:
Archived threads are classified separately from normal threads, so both need to be looped over. The two for loops also loop over different things. I had them separate since `channel.archived_threads()` is an async method that yields a generator while `channel.threads` is a list.  To my knowledge, I believe it'd be more effort than it's worth to combine the two if it is even possible.

Reply:
it looks like they have the same content though. we're passing the threads to `automod_thread` the exact same way and the function seems like it doesnt distinguish between the two different kinds of threads. you could do `threads = channel.archived_threads + channel.threads` and it should automatically combine the two lists

Reply:
channel.archived_threads() does not give a list so I can't do that
```
TypeError: unsupported operand type(s) for +: 'async_generator' and 'list'
```

Reply:
Ew okay I might poke at it on my own but we can put this in for now